CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ yarn build
 Run Storybook.
 
 ```shell
-yarn storybook
+yarn start
```

Comment:
Why `yarn storybook` doesn't work? 🤔 I thought this was a default command.

Reply:
Not sure but `yarn start-storybook` works